content/06.TechnicalValidation.md

```diff
@@ -5,17 +5,17 @@ Each individual dataset also has individual vegetation plots, each provided by i
 We obviously have no direct control on the individual vegetation plots that we provide here in an open access dataset. 
 Yet, each of these vegetation plots are stemming from trained professional botanists, or published scientific work, and are accompanied by detailed information on the sampling protocols used, thus ensuring data quality and reliability.  
 
-Before having been integrated into the sPlot database, each dataset was further checked for consistency and converted to Turboveg format (CIT). 
-During this conversion into a Turboveg format, we checked that all dataset contained the required metadata information and we converted this information to the sPlot database standards, if necessary. 
+Before having been integrated into the sPlot database, each dataset was further checked for consistency and, if having a different format, was converted to a Turboveg 2 database (Hennekens & Schaminée 2001). 
```

Comment:
To have the citation properly formatted you can cite using the tag 
"@doi:"
in this case: @doi:10.2307/3237010 for Hennekens & Schaminee 2001 and @doi:10.5281/zenodo.845445
I'll take care of that